src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -73,12 +73,14 @@ public static bool IsCloseParenInParameterList(this SyntaxToken token)
 
         public static bool IsOpenParenInArgumentList(this SyntaxToken token)
         {
-            return token.Kind() == SyntaxKind.OpenParenToken && token.Parent.Kind() == SyntaxKind.ArgumentList;
+            return token.Kind() == SyntaxKind.OpenParenToken &&
+                (token.Parent.IsKind(SyntaxKind.ArgumentList) || token.Parent.IsKind(SyntaxKind.AttributeArgumentList));
         }
```

Comment:
You just made a change to deal Parent being null.  Should you use `IsParentKind()` to handle that case?